pkg/apis/pipeline/v1alpha1/task_types.go

```diff
@@ -104,11 +110,13 @@ type Inputs struct {
 	// PipelineResources.
 	// +optional
 	Resources []TaskResource `json:"resources,omitempty"`
-	// Params is a list of input parameters required to run the task. Params
+
+	// Depraceted
+	// DeprecatedParams is a list of input parameters required to run the task. Params
 	// must be supplied as inputs in TaskRuns unless they declare a default
 	// value.
 	// +optional
-	Params []ParamSpec `json:"params,omitempty"`
+	DeprecatedParams []ParamSpec `json:"params,omitempty"`
```

Comment:
Renaming it makes it easier to know it's deprecated and we shouldn't depend/looking at it in the reconcilier